doc/setup/email_gateway.rst

```diff
@@ -0,0 +1,40 @@
+:banner: banners/email_gateway.jpg
+
+==================
+Odoo email gateway
+==================
+
+The Odoo mail gateway is a better alternative than using an "incoming mail server"
+because the messages are received instantaneously.
+
+The principle is straightforward. Your incoming SMTP server will receive emails and directly forward
+them through a script (``odoo/addons/mail/static/scripts/odoo-mailgate.py``).
+
+This script will simply connect to your database through XMLRPC to send the mails to MailThread.message_process().
+
+Prerequisites
+-------------
+- Administrator access to the database
+- Technical knowledge on how to configure a email server
+- Your own mail server such as Postfix or Exim
```

Comment:
- Administrator access to the Odoo database.
- Your own mail server such as Postfix or Exim.
- Technical knowledge on how to configure an email server.